doc/proposals/TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+## <Title of Proposal>
+
```

Comment:
Should we also track status? I can imagine:
* Draft
* Accepted
* Implemented
* Abandoned

Reply:
Yes that would be good, or maybe also move it to different directories? 

Currently, I went with what we already had in place, by looking at our current proposal docs but would I actually ~steal~ borrow from [this](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/Design_Proposal_TEMPLATE.md) if everyone agrees?

Reply:
Draft should be a PR no? and Abandoned should just be removed IMO. thoughts?

Reply:
> Draft should be a PR no?

Agreed. If we include the status in the template, I would think the template would just have "Status: Draft" hardcoded.

> Abandoned should just be removed IMO.

Possibly, but if there are details in the proposal about why it was abandoned, it might make sense to keep it around for posterity.

Reply:
you can also find this with git or the initial proposal PR right?